packages/core/src/Footer/Footer.stories.tsx

```diff
@@ -13,8 +13,8 @@ export default meta;
 
 export const Main: StoryObj<HvFooterProps> = {
   args: {
-    name: "Hitachi Vantara",
-    copyright: "© Hitachi Vantara Corporation 2022",
+    name: "Pentaho+",
+    copyright: "© Hitachi Vantara Corporation 2024",
```

Comment:
I think we should remove the `args` instead. This way, each theme would get their default `name`/`copyright` values (which we can more easily test against), and Storybook users can still override the props